docs/docker/README.md

```diff
@@ -18,6 +18,23 @@ Run forever as a background service, listen on port `53/udp` on all network inte
 $ docker run --detach --restart unless-stopped --net=host commonshost/dohnut --listen 0.0.0.0:53 --doh commonshost --bootstrap 1.1.1.1
 ```
 
+Or using docker-compose:
+```yaml
+version: "3"
+
+services:
+  dohnut:
+    container_name: dohnut
+    image: commonshost/dohnut:latest
+    environment:
+      TZ: "Europe/Brussels"
+      DOHNUT_LISTEN: 127.0.0.1:53
+      DOHNUT_BOOTSTRAP: 1.1.1.1
```

Comment:
Did you mean to add multiple IPs here?

Reply:
In the documentation I didn't wanted to follow the above example which only used 1 ip. But you definitely can use multiple ip's when the fix is applied.